src/components/FileTreeNode/index.spec.tsx

```diff
@@ -45,17 +50,42 @@ describe(__filename, () => {
 
   const render = ({
     version = createInternalVersion(fakeVersion),
+    linterMessages,
     ...props
   }: RenderParams = {}) => {
     const allProps = {
       version,
+      linterMessages,
       ...getTreefoldRenderProps(),
       ...props,
     };
 
     return shallow(<FileTreeNode {...allProps} />);
   };
 
+  const _getMessageMap = (
+    messages: Partial<ExternalLinterMessage>[] = [fakeExternalLinterMessage],
+  ) => {
+    return getMessageMap(createFakeExternalLinterResult({ messages }));
+  };
+
+  const renderWithLinterMessage = ({
+    version = createInternalVersion(fakeVersion),
+    message = fakeExternalLinterMessage,
```

Comment:
This seems like it would be better as an array but I guess if you don't need it then it's fine.

Reply:
yep, I'll do it in the next PR